src/celscript/lua/celx_internal.h

```diff
@@ -45,6 +45,10 @@ enum
     Celx_Category = 13
 };
 
+template<typename T> int celxClassId(T)
```

Comment:
i have no idea how it worked, but with Image moved to celestia::engine g++ is no more able find a proper definition.